packages/scss/src/components/_button.scss

```diff
@@ -398,91 +402,120 @@
 // WRAPPER
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
+ul.button-group {
+	flex-wrap: wrap;
+
+	.button.mod-more {
+		height: 100%;
+		min-height: calc(#{_theme('spacings.big')} + #{_theme('spacings.smaller')});
```

Comment:
Same, not sure it's the way spacings were designed as users can override these properties in order to edit margins & paddings